cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/AbstractProgramRuntimeService.java

```diff
@@ -118,7 +118,7 @@ public final RuntimeInfo run(ProgramDescriptor programDescriptor, ProgramOptions
       return runtimeInfo;
     } catch (Exception e) {
       cleanUpTask.run();
-      LOG.error("Exception while trying to createPluginSnapshot", e);
+      LOG.error("Exception while trying to run program", e);
```

Comment:
I'm not sure why it was done that way.
It's possible that the calling code doesn't log it the same way (since its from http handler).